neat/tools/fittools/compartmentfitter.py

```diff
@@ -181,12 +164,15 @@ def setName(self, name, path):
         self.name = name
         self.path = path
 
-    def setImpedancesInTree(self, many_freqs=False, recompute=False, pprint=False):
+    def setImpedancesInTree(self, sv_h=None, many_freqs=False, recompute=False, pprint=False):
```

Comment:
ouch, API change needs to be highlighted or is this never called by the user?

Reply:
This is not an API function, only the methods of `CompartmentFitter` are.